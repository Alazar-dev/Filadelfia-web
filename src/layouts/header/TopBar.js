import { Box, Typography, Grid, List, Divider } from "@material-ui/core";
import { makeStyles } from "@material-ui/core/styles";

import Twitter from "../../assets/Header/Twitter.png";
import Google from "../../assets/Header/Google Plus.png";
import Facebook from "../../assets/Header/Facebook.png";
import Linkedin from "../../assets/Header/Linked IN.png";
import Phone from "../../assets/Header/Icon Phone.png";
import Location from "../../assets/Header/Icon Location.png";
import Web from "../../assets/Header/Icon Web.png";
const useStyles = makeStyles({
  root: {
    height: "45px",
    "@media (max-width: 900px)": {
      // display: 'none',
      display: "none",
    },
  },
  socialLinks: {
    display: "flex",
    alignItems: "center",
    paddingRight: "24px",
  },
  link: {
    marginLeft: 20,
  },
  item: {
    paddingTop: "0px !important",
    paddingBottom: "0px !important",
    display: "flex",
    alignItems: "center",
  },
  text: {
    textAlign: "left",
    letterSpacing: "0.14px",
    color: "#343C99",
    opacity: 1,
  },
  grid: {
    margin: 0,
    width: "auto",
  },
});

const TopBar = () => {
  const classes = useStyles();
  return (
    <Box className={classes.root}>
      <Grid justify="center" spacing={6} className={classes.grid} container>
        <Divider orientation="vertical" flexItem />
        <Grid className={classes.item} item>
          <Typography className={classes.text} variant="subtitle2">
            <img src={Phone} alt="" /> +251 011 657 4000
          </Typography>
        </Grid>
        <Divider orientation="vertical" flexItem />
        <Grid className={classes.item} item>
          <Typography className={classes.text} variant="subtitle2">
            +251 011 657 4001
          </Typography>
        </Grid>
        <Divider orientation="vertical" flexItem />
        <Grid className={classes.item} item>
          <Grid spacing={2} container direction="row" alignItems="center">
            <Grid item>
              <img src={Web} alt="" />
            </Grid>
            <Grid item>
              <Typography className={classes.text} variant="subtitle2">
                info@Phillipisiyoscurch.com
              </Typography>
            </Grid>
          </Grid>
        </Grid>
        <Divider orientation="vertical" flexItem />
        <Grid className={classes.item} item>
          <Grid spacing={2} container direction="row" alignItems="center">
            <Grid item>
              <img src={Location} alt="" />
            </Grid>
            <Grid item>
              <Typography className={classes.text} variant="subtitle2">
                Bole Michael, infront of Miachel Church
              </Typography>
            </Grid>
          </Grid>
        </Grid>
        <Divider orientation="vertical" flexItem />{" "}
        <Grid className={classes.item} item>
          <List className={classes.socialLinks}>
            <a href="https://www.twitter.com">
              <img className={classes.link} src={Twitter} alt="twitter" />
            </a>
            <a href="https://www.googleplus.com">
              <img className={classes.link} src={Google} alt="google plus" />
            </a>
            <a href="https://www.facebook.com">
              <img className={classes.link} src={Facebook} alt="facebook" />
            </a>
            <a href="https://www.linkedin.com">
              <img className={classes.link} src={Linkedin} alt="linkedin" />
            </a>
          </List>
        </Grid>
        <Divider orientation="vertical" flexItem />
      </Grid>
    </Box>
  );
};

export default TopBar;
