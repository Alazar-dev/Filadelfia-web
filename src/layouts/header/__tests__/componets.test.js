import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import Appbar from "../AppBar";
import TopBar from "../TopBar";

describe("Rendering Navbar", () => {
  it("renders a Appbar", () => {
    const tree = renderer.create(<Appbar />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders a TopBar", () => {
    const tree = renderer.create(<TopBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
