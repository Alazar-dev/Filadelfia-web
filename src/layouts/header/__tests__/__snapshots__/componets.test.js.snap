// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Navbar renders a Appbar 1`] = `
<header
  className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorDefault makeStyles-header-1 MuiPaper-elevation4"
>
  <div
    className="MuiBox-root MuiBox-root-13 makeStyles-root-7"
  >
    <div
      className="MuiGrid-root makeStyles-grid-12 MuiGrid-container MuiGrid-spacing-xs-6 MuiGrid-justify-xs-center"
    >
      <hr
        className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
      />
      <div
        className="MuiGrid-root makeStyles-item-10 MuiGrid-item"
      >
        <h6
          className="MuiTypography-root makeStyles-text-11 MuiTypography-subtitle2"
        >
          <img
            alt=""
            src="Icon Phone.png"
          />
           +251 011 657 4000
        </h6>
      </div>
      <hr
        className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
      />
      <div
        className="MuiGrid-root makeStyles-item-10 MuiGrid-item"
      >
        <h6
          className="MuiTypography-root makeStyles-text-11 MuiTypography-subtitle2"
        >
          +251 011 657 4001
        </h6>
      </div>
      <hr
        className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
      />
      <div
        className="MuiGrid-root makeStyles-item-10 MuiGrid-item"
      >
        <div
          className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <img
              alt=""
              src="Icon Web.png"
            />
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <h6
              className="MuiTypography-root makeStyles-text-11 MuiTypography-subtitle2"
            >
              info@Phillipisiyoscurch.com
            </h6>
          </div>
        </div>
      </div>
      <hr
        className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
      />
      <div
        className="MuiGrid-root makeStyles-item-10 MuiGrid-item"
      >
        <div
          className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <img
              alt=""
              src="Icon Location.png"
            />
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <h6
              className="MuiTypography-root makeStyles-text-11 MuiTypography-subtitle2"
            >
              Bole Michael, infront of Miachel Church
            </h6>
          </div>
        </div>
      </div>
      <hr
        className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
      />
       
      <div
        className="MuiGrid-root makeStyles-item-10 MuiGrid-item"
      >
        <ul
          className="MuiList-root makeStyles-socialLinks-8 MuiList-padding"
        >
          <a
            href="https://www.twitter.com"
          >
            <img
              alt="twitter"
              className="makeStyles-link-9"
              src="Twitter.png"
            />
          </a>
          <a
            href="https://www.googleplus.com"
          >
            <img
              alt="google plus"
              className="makeStyles-link-9"
              src="Google Plus.png"
            />
          </a>
          <a
            href="https://www.facebook.com"
          >
            <img
              alt="facebook"
              className="makeStyles-link-9"
              src="Facebook.png"
            />
          </a>
          <a
            href="https://www.linkedin.com"
          >
            <img
              alt="linkedin"
              className="makeStyles-link-9"
              src="Linked IN.png"
            />
          </a>
        </ul>
      </div>
      <hr
        className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
      />
    </div>
  </div>
  <div
    className="makeStyles-navbar-5"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-3 MuiToolbar-gutters"
    >
      <div
        className="makeStyles-logo-6"
      >
        <img
          alt=""
          src="Logo.png"
        />
      </div>
      <div>
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-menuButton-2 MuiButton-colorInherit"
          href="/"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Home
          </span>
        </a>
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-menuButton-2 MuiButton-colorInherit"
          href="/sermons"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Sermons
          </span>
        </a>
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-menuButton-2 MuiButton-colorInherit"
          href="/prayerwall"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Prayer Wall
          </span>
        </a>
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-menuButton-2 MuiButton-colorInherit"
          href="/events"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Events
          </span>
        </a>
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-menuButton-2 MuiButton-colorInherit"
          href="/gallery"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Gallery
          </span>
        </a>
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-menuButton-2 MuiButton-colorInherit"
          href="/contact"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Contact
          </span>
        </a>
      </div>
    </div>
  </div>
</header>
`;

exports[`Rendering Navbar renders a TopBar 1`] = `
<div
  className="MuiBox-root MuiBox-root-14 makeStyles-root-7"
>
  <div
    className="MuiGrid-root makeStyles-grid-12 MuiGrid-container MuiGrid-spacing-xs-6 MuiGrid-justify-xs-center"
  >
    <hr
      className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
    />
    <div
      className="MuiGrid-root makeStyles-item-10 MuiGrid-item"
    >
      <h6
        className="MuiTypography-root makeStyles-text-11 MuiTypography-subtitle2"
      >
        <img
          alt=""
          src="Icon Phone.png"
        />
         +251 011 657 4000
      </h6>
    </div>
    <hr
      className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
    />
    <div
      className="MuiGrid-root makeStyles-item-10 MuiGrid-item"
    >
      <h6
        className="MuiTypography-root makeStyles-text-11 MuiTypography-subtitle2"
      >
        +251 011 657 4001
      </h6>
    </div>
    <hr
      className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
    />
    <div
      className="MuiGrid-root makeStyles-item-10 MuiGrid-item"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <img
            alt=""
            src="Icon Web.png"
          />
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <h6
            className="MuiTypography-root makeStyles-text-11 MuiTypography-subtitle2"
          >
            info@Phillipisiyoscurch.com
          </h6>
        </div>
      </div>
    </div>
    <hr
      className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
    />
    <div
      className="MuiGrid-root makeStyles-item-10 MuiGrid-item"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <img
            alt=""
            src="Icon Location.png"
          />
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <h6
            className="MuiTypography-root makeStyles-text-11 MuiTypography-subtitle2"
          >
            Bole Michael, infront of Miachel Church
          </h6>
        </div>
      </div>
    </div>
    <hr
      className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
    />
     
    <div
      className="MuiGrid-root makeStyles-item-10 MuiGrid-item"
    >
      <ul
        className="MuiList-root makeStyles-socialLinks-8 MuiList-padding"
      >
        <a
          href="https://www.twitter.com"
        >
          <img
            alt="twitter"
            className="makeStyles-link-9"
            src="Twitter.png"
          />
        </a>
        <a
          href="https://www.googleplus.com"
        >
          <img
            alt="google plus"
            className="makeStyles-link-9"
            src="Google Plus.png"
          />
        </a>
        <a
          href="https://www.facebook.com"
        >
          <img
            alt="facebook"
            className="makeStyles-link-9"
            src="Facebook.png"
          />
        </a>
        <a
          href="https://www.linkedin.com"
        >
          <img
            alt="linkedin"
            className="makeStyles-link-9"
            src="Linked IN.png"
          />
        </a>
      </ul>
    </div>
    <hr
      className="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
    />
  </div>
</div>
`;
